packages/dx-react-grid-demos/index.html

```diff
@@ -8,7 +8,7 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 </head>
 <body>
-  <div class="embedded-demo" data-options='{ "router": "hash", "showThemeSelector": true }'>Loading...</div>
+  <div class="embedded-demo" data-options='{ "router": "hash", "showThemeSelector": true, "frame": true }'>Loading...</div>
```

Comment:
Why are you changing the `frame` parameter here?

Reply:
Mistake